src/pybind/mgr/balancer/module.py

```diff
@@ -697,8 +697,13 @@ def do_crush_compat(self, plan):
         osdmap = ms.osdmap
         crush = osdmap.get_crush()
         pe = self.calc_eval(ms, plan.pools)
-        if pe.score == 0:
-            detail = 'Distribution is already perfect'
+        min_score_to_optimize = float(self.get_config('min_score', 0))
```

Comment:
should we make this .001 or something?

Reply:
I was thinking about that value too. But user can manually configure that anyway.